core/subscriptions.go

```diff
@@ -26,8 +26,6 @@ func subscribe(channel string, client *client) {
 func unsubscribe(channel string, client *client) {
 	filtered := clients{}
 
-	// Probably can't handle unsubscribing from a non-existent channel, but
```

Comment:
because it is usless to check that. Unsubscribe method checks if person is unsubscribing from not subscribed channel, never ends up there